dev/devicelab/bin/tasks/gradle_plugin_light_apk_test.dart

```diff
@@ -325,22 +313,6 @@ Future<void> main() async {
               result);
       });
 
-      await runPluginProjectTest((FlutterPluginProject pluginProject) async {
```

Comment:
I think this one is already covered in https://github.com/flutter/flutter/blob/e8dc7a2e172d8a6eac23fb47716fc2142d4e388b/dev/devicelab/bin/tasks/gradle_plugin_fat_apk_test.dart#L16-L28